yarn.lock

```diff
@@ -5,7 +5,6 @@
 "@gulp-sourcemaps/identity-map@1.X":
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/@gulp-sourcemaps/identity-map/-/identity-map-1.0.2.tgz#1e6fe5d8027b1f285dc0d31762f566bccd73d5a9"
-  integrity sha512-ciiioYMLdo16ShmfHBXJBOFm3xPC4AuwO4xeRpFeHz7WK9PYsWCmigagG2XyzZpubK4a3qNKoUBDhbzHfa50LQ==
```

Comment:
I noticed this update is removing the integrity checksums for the yarn dependencies, which suggests you might be using an older version of Yarn (I think?).

Try running `brew upgrade yarn` to install the latest version, then delete `node_modules`, `yarn.lock` and trying installing via `yarn` again?